### PR TITLE
Soap test 01

### PR DESCRIPTION
**Pull request by UnixJunkie**
Librascal beginner trying to dump SOAP features of atoms in molecules

### CODE REVIEW

File: soap_test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "gaussian_sigma_constant": 0.5,
+    "soap_type": soap_type,
+    # 'RadialScaling' is the only other choice
+    "cutoff_function_type": "ShiftedCosine",
+    "normalize": True, # recommended
+    # 'DVR' is the only other choice
+    "radial_basis": "GTO",
+    # # as far as I can tell, global_species has no effect on fixing the
+    # # dimensionality of the SOAP feature vectors
+    # "global_species": global_species
```
Comment (Luthaf) [inline soap_test.py:47]: Try also specifying `expansion_by_species_method`:

```suggestion
    "global_species": global_species,
    "expansion_by_species_method": "user defined",
```

It should not be required but something might have felt between the cracks.

File: soap_test.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Try to dump SOAP vectors (features) for each atom of each molecule
+
+import sys
+import numpy as np
+from ase.io import read as ase_read
+from rascal.representations import SphericalInvariants as SOAP
+
+# What is this integer? Unit? Reasonable range to explore?
```
Comment (Luthaf) [inline soap_test.py:10]: that's the number of radial basis, i.e. the radial resolution. Typical range is 4-9, GTO can go up to 22, DVR is mostly unlimited.

File: soap_test.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+max_radial = 6
+# What is this integer? Unit? Reasonable range to explore?
```
Comment (Luthaf) [inline soap_test.py:12]: that's the number of spherical harmonics being used for the basis, i.e. the angular resolution. Typical range is 4-8, you can go higher if you want/need

File: soap_test.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+max_angular = 6
+# what is a power spectrum?
```
Comment (Luthaf) [inline soap_test.py:14]: The power spectrum is the standard SOAP descriptor, but there are many other that can be constructed. Power spectrum is a 3-body descriptor, while the radial spectrum is a 2-body representation. 

Bi-Spectrum is the name of the 4-body representation, and then we just refer to them with the body order.

File: soap_test.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    "interaction_cutoff": 3.5,
+    # Angstroms? what is the reasonable range to explore for this value?
+    "cutoff_smooth_width": 0.5,
+    "max_radial": max_radial,
+    "max_angular": max_angular,
+    "gaussian_sigma_type": "Constant", # only possible choice
+    # Angstroms? what is the reasonable range to explore for this value?
+    "gaussian_sigma_constant": 0.5,
+    "soap_type": soap_type,
+    # 'RadialScaling' is the only other choice
```
Comment (Luthaf) [inline soap_test.py:40]: RadialScaling is defined in https://pubs.rsc.org/en/content/articlelanding/2018/CP/C8CP05921G, for the QM9 example

**Comment by UnixJunkie** [comment]
Indeed, the combination of both 
```python
"global_species": global_species,
"expansion_by_species_method": "user defined",
```
fixes the length of the SOAP vectors.
Thanks a lot!

**Pull request closed without merging**